qa/update-tests/src/test/java/io/camunda/zeebe/test/ContainerState.java

```diff
@@ -68,12 +70,18 @@ final class ContainerState implements CloseableResource {
   private DockerImageName gatewayImage;
   private boolean withRemoteDebugging;
 
+  private String withUser;
+
   ZeebeClient client() {
     return client;
   }
 
   ContainerState withOldBroker() {
-    return broker(PREVIOUS_VERSION);
+    broker(PREVIOUS_VERSION);
+    // user needs to be set to allow a smooth update from zeebe 8.2 to 8.3
+    // as the default user changed to `zeebe` with 8.3 and was `root` with 8.2
+    withUser(ZEEBE_USER);
```

Comment:
:upside_down_face: We could put a TODO to remove this post 8.3 release on main (i.e. with 8.4.0-SNAPSHOT). At the same time, it doesn't really hurt I guess :shrug:

Reply:
also created follow-up task https://github.com/camunda/zeebe/issues/13528